src/verifytime.h

```diff
@@ -12,6 +12,11 @@
 extern "C" {
 #endif
 
+/**
+ * @brief Character used as PATH separator.
+ */
+#define PATH_SEPARATOR '/'
```

Comment:
this shouldn't be exported. define in the .c file

Reply:
I can do that. But I'd like to understand this better, can you please explain me why this should not be exported?

Reply:
On header files we export everything that we want others to access and use. Things that we don't want others to use, we should keep only in the C file. As this is a header for a library to verify time, I don't see a reason on exporting a path separator.

Besides that we already export a constant with the same name in other header file. In this case the value is also identical, but if for some reason it is changed we would have files that import both headers with different values for the same constant.

Reply:
I see. Thanks for explaining that :)